tests/metrics/test_axiomatic_metrics.py

```diff
@@ -4,12 +4,12 @@
 import pytest
 from pytest_lazyfixture import lazy_fixture
 
-from ..fixtures import *
-from ...quantus.metrics import *
-from ...quantus.helpers import *
-from ...quantus.helpers.explanation_func import explain
-from ...quantus.helpers.pytorch_model import PyTorchModel
-from ...quantus.helpers.tf_model import TensorFlowModel
+from tests.fixtures import *
```

Comment:
Should we replace the * here with more precise imports of functions?

Reply:
yes, did that everywhere it made sense. specifically the fixtures i left as *, since all fixtures should be available for tests i think

Reply:
also, e.g. in test_similarity_func i imported similarity funcs with *, because the tests need access to all similarity functions

Reply:
i think at least `from quantus.metrics.axiomatic import *`
should be replaced with something this:
```
from quantus.metrics.axiomatic import Completeness
from quantus.metrics.axiomatic import NonSensitivity
from quantus.metrics.axiomatic import InputInvariance
```

Regarding the fixture import I don't know how to do this efficiently without the star.

Reply:
should we maybe do this (similarly in all the other metrics folders):

from quantus.metrics.axiomatic import (Completeness, NonSensitivity, InputInvariance)

Reply:
changed this. The fixtures remain as * imports though.